packages/muelu/src/Graph/PairwiseAggregation/MueLu_NotayAggregationFactory_def.hpp

```diff
@@ -208,14 +206,12 @@ namespace MueLu {
       orderingVector[i] = i;
     if (ordering == O_RANDOM)
       MueLu::NotayUtils::RandomReorder(orderingVector);
-#if defined(HAVE_MUELU_KOKKOS_REFACTOR)
```

Comment:
As explained above and as is clear from `MueLu::Utilities_kokkos` the Cuthill-McKee reordering needs to be guarded with appropriate `HAVE_MUELU_KOKKOS_REFACTOR`